src/Shared/DashboardConfigNames.cs

```diff
@@ -12,6 +12,7 @@ internal static class DashboardConfigNames
     public static readonly ConfigName DashboardConfigFilePathName = new("DOTNET_DASHBOARD_CONFIG_FILE_PATH");
     public static readonly ConfigName ResourceServiceUrlName = new("DOTNET_RESOURCE_SERVICE_ENDPOINT_URL");
 
+    public static readonly ConfigName DashboardFileConfigDirectory = new("Dashboard:FileConfigDirectory", "DASHBOARD__FILECONFIGDIRECTORY");
```

Comment:
This config name scheme (`Dashboard:xxx:yyy`) is for binding onto the dashboard options. That's not happening here so this scheme shouldn't be used. The name should be like `ASPNETCORE_URLS`, `DOTNET_DASHBOARD_CONFIG_FILE_PATH`, etc.

I think it should be `DOTNET_DASHBOARD_CONFIG_FILE_DIRECTORY`. It's close to the file path config config value, but we can't go back in time and add JSON to the existing config name so we'll have to live with it.

Reply:
Issue: https://github.com/dotnet/aspire/issues/5760